src/header.jl

```diff
@@ -132,7 +132,13 @@ function path_header(sys_path::AbstractString, tar_path::AbstractString)
     elseif isfile(st)
         size = filesize(st)
         type = :file
-        mode = iszero(filemode(st) & 0o100) ? 0o644 : 0o755
+        # Windows can't re-use the `lstat` result, we need to ask
+        # the system whether it's executable or not via `access()`
+        if Sys.iswindows()
+            mode = Sys.isexecutable(sys_path) ? 0o755 : 0o644
```

Comment:
I was under the impression that you change caused `mode` to return something that reflects the executability of the file on Windows. Is that not the case?

Reply:
No from “stat”, sadly. Only from isexecutable. It is possible to have stat do something similar, but it’s more involved.